test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -82,7 +86,7 @@ def convert_to_onnx(model, input=None, opset_version=9, do_constant_folding=True
                     keep_initializers_as_inputs=True, dynamic_axes=None,
                     input_names=None, output_names=None,
                     fixed_batch_size=False, training=None,
-                    verbose=False):
+                    verbose=False, operator_export_type=torch.onnx.OperatorExportTypes.ONNX):
```

Comment:
Add `from torch.onnx import OperatorExportTypes` at the top to make other references to it much shorter.